ports/chroma/port.json

```
@@ -0,0 +1,40 @@
+{
+    "version": 3,
+    "name": "chroma.zip",
+    "items": [
+        "Chroma.sh",
+        "chroma"
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "Chroma",
+        "porter": [
+            "Cebion"
+        ],
+        "desc": "Chroma is an abstract puzzle game. A variety of colourful shapes are arranged in a series of increasingly complex patterns, forming fiendish traps that must be disarmed and mysterious puzzles that must be manipulated in order to give up their subtle secrets. Initially so straightforward that anyone can pick it up and begin to play, yet gradually becoming difficult enough to tax even the brightest of minds.",
+        "desc_md": null,
+        "inst": "Ready to run!",
+        "inst_md": null,
+        "genres": [
+            "puzzle"
+        ],
+        "image": null,
+        "rtr": true,
+        "exp": false,
+        "runtime": null,
+        "reqs": [],
+        "arch": [
+            "aarch64"
+        ]
+    },
+    "status": {
+        "source": "Unknown",
+        "md5": null,
+        "status": "Installed"
+    },
+    "files": {
+        "port.json": "chroma/port.json",
+        "Chroma.sh": "Chroma.sh",
+        "chroma": "chroma"
+    }
```

I think it needs to be removed.
```suggestion
    }
```